content/terms_and_definitions/terms_and_definitions.adoc

```diff
@@ -6,7 +6,86 @@ ifndef::include-only-once[]
 include::{root-path}_config.adoc[]
 endif::[]
 
-TODO: Add terms and definitions
+3D Asset::
+A 3D model. containing the 3D geometry, combined with assigned material properties and specific meta data.
 
-Static road network::
-Collection of connected roads enriched by static objects that do not change during runtime of a simulation.
+3D Model::
+A (mathematical) geometrical, coordinate-based representation of the surface of an object.
+This excludes physical material properties. For a combination of 3D geometry and physical material properties, see 3D Asset.
+
+3D Environment::
+A 3D representation of the surrounding of a simulated sensor.
+The 3D environment can consist of multiple 3D assets.
+
+Co-Simulation::
+Simulating multiple independent models connected with interfaces.
+A co-simulation platform handles the timing and data transfer between the individual models
+
+Ground truth::
+True static and dynamic states of all objects as defined by content creator or observed by omniscient observer (e.g. simulation platform).
+
+Material::
+A collection of material properties belonging to one entity.
+
+Material assignment::
+A mapping of material IDs to parts of 3D models.
+The material IDs are unique a point to a set of material properties in separate material files.
```

Comment:
```suggestion
The material IDs are a unique pointer to a set of material properties in separate material files.
```

Reply:
Shouldn't it be "A material ID is a unique pointer ..." or "The material IDs are unique pointers ..."?

Reply:
Yes, thanks for the remark. I updated the suggestion

Reply:
done